test/spec/controllers/main.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+describe('Controller: MainCtrl', function () {
+
+  beforeEach(module('todoChallengeApp'), module('LocalStorageModule'));
+
+  var MainCtrl,
+    scope;
+
+  beforeEach(inject(function ($controller, $rootScope) {
+    scope = $rootScope.$new();
+    MainCtrl = $controller('MainCtrl', {
+      $scope: scope
+    });
+  }));
+
+  it('should have no items to start', function () {
+    expect(scope.todos.length).toBe(0);
+  });
+
+  it('should add items to the list', function () {
+    scope.todo = 'Testing';
+    scope.addTodo();
+    expect(scope.todos.length).toBe(1);
+  });
+
+  it('should add then remove an item from the list', function () {
+    scope.todo = 'Testing';
+    scope.addTodo();
+    expect(scope.todos.length).toBe(1);
```

Comment:
This is a repeat of the previous test.. if that passes, you can assume that it'll pass on this bit. Test only what the current test is testing